packages/melos/CHANGELOG.md

```diff
@@ -367,7 +367,7 @@ prevent a commit from being automatically created after versioning.
 
 ## 1.5.0
 
- - **FIX**: export `MelosConfigException` for programatic usage. ([8b7fbfb5](https://github.com/invertase/melos/commit/8b7fbfb55ea223c11da370e4b2b3feb57c347f20))
+ - **FIX**: export `MelosConfigException` for programmatic usage. ([8b7fbfb5](https://github.com/invertase/melos/commit/8b7fbfb55ea223c11da370e4b2b3feb57c347f20))
```

Comment:
Changelog lines should never change even though there are mistakes in them, the last change of them should be connected to the release commit they were included in.